devimint/src/vars.rs

```diff
@@ -125,6 +125,7 @@ declare_vars! {
 
         FM_GATEWAY_API_ADDR: String = f!("http://127.0.0.1:{FM_PORT_GW_CLN}");
         FM_GATEWAY_PASSWORD: String = "theresnosecondbest";
+        FM_GATEWAY_FEES: String = "0,0"; // Enable to us to make an unbounded number of payments
```

Comment:
Not ideal to not test fees anymore, but ok.

Reply:
It makes sense for automated tests but it doesn't make sense for manual tests